models/listing.js

```diff
@@ -1,11 +1,26 @@
 var mongoose = require('mongoose');
 var FeedItem = mongoose.model('feeditems');
 
+
+// Create productVariant schema to be a subdocument array in listing
+// http://mongoosejs.com/docs/subdocs.html
+var productVariantSchema = new mongoose.Schema({
+    vendorId: Number,
+    title: String,
+    sku: String,
+    option1: String,
+    option2: String,
+    option3: String,
+    imageId: Number,
+    inventoryQuantity: Number
```

Comment:
How do we want the client to know whether to render "sold out" or not? Sum the quantity of all variants?